apps/backend/src/app/api/latest/integrations/neon/projects/transfer/confirm/verification-code-handler.tsx

```diff
@@ -25,6 +25,27 @@ export const neonIntegrationProjectTransferCodeHandler = createVerificationCodeH
       project_id: yupString().defined(),
     }).defined(),
   }),
+  async validate(tenancy, method, data, body, user) {
+    const project = tenancy.project;
+    if (project.id !== "internal") throw new StatusError(400, "This endpoint is only available for internal projects.");
+    if (!user) throw new KnownErrors.UserAuthenticationRequired;
```

Comment:
We need this in `handle`, but not `validate` (you should be able to validate an endpoint without being logged in)

Reply:
```suggestion
```